pkg/git/git_test.go

```diff
@@ -64,6 +67,84 @@ func TestGetDefaultKubernetesRepoURLSuccess(t *testing.T) {
 	}
 }
 
+// createTestRepository creates a test repo, cd into it and returns the path
+func createTestRepository() (repoPath string, err error) {
+	repoPath, err = ioutil.TempDir(os.TempDir(), "sigrelease-test-repo-*")
+	if err != nil {
+		return "", errors.Wrap(err, "creating a directory for test repository")
+	}
+	if err := os.Chdir(repoPath); err != nil {
+		return "", errors.Wrap(err, "cd'ing into test repository")
+	}
+	out, err := exec.Command("git", "init").Output()
+	if err != nil {
+		return "", errors.Wrapf(err, "initializing test repository: %s", out)
+	}
+	return repoPath, nil
+}
+
+func TestGetUserName(t *testing.T) {
+	const fakeUserName = "SIG Release Test User"
+	currentDir, err := os.Getwd()
+	if err != nil {
+		t.Fatal("error reading the current directory")
```

Comment:
Let’s use testify’s `require.Nil()` for those checks. Saves two lines of code.

Reply:
done